src/credentials/loginManager.ts

```diff
@@ -100,5 +106,6 @@ export class LoginManager {
      */
     public async logout(): Promise<void> {
         await this.awsContext.setCredentials(undefined)
+        vscode.commands.executeCommand('setContext', CONTEXT_KEY_ISCONNECTED, false)
```

Comment:
I guess we would also need to handle casese where the session timed out.

But I'm having second guesses about this, it looks too complicated. Maybe we should just rename the `Connect to AWS` command to `Connect AWS Profile...`

Reply:
Considering that 'Connect to AWS' and 'Switch Profile' are literally the same command, it does make sense to just name it something that accounts for both scenarios.

Reply:
Does that give a sense that a user can connect with more than one profile at a time? I'm not worried about the complexity (especially if we contain all login/logout logic to the same entrypoint).

Reply:
> Does that give a sense that a user can connect with more than one profile at a time?

what about `Choose AWS Profile...` ?

Reply:
or maybe `Connect to AWS Profile...`  ?

Reply:
The verbs `Select` or `Choose` would probably be best since we are prompting from a fixed set of profiles.